changelog/changes/Bf9B2A5wAKlHFZWTi4EEwa7yAE.md

```diff
@@ -0,0 +1,9 @@
+---
+title: "Correctly update aggregations in `chart` operators"
```

Comment:
Unlike PRs, changelog entries should not use the imperative voice.

Reply:
Reworded the entry.